convert2rhel/pkghandler.py

```diff
@@ -86,49 +86,47 @@
         "packager",
         "vendor",
         "nevra",
-        "fingerprint",
```

Comment:
Is this correct?

Reply:
Since this is just a structure to hold the information that comes from the rpm --qf command, I thought that maintaining the key_id here instead of fingerprint would be good. 

The ticket mentions that the occurrence of all fingerprint mentions should be replaced to key_id, so, this is not different. 

Also, the legacy API also has it changed from fingerprint to key_id